ui-tests/src/test/java/org/openmrs/reference/page/HomePage.java

```diff
@@ -43,7 +44,7 @@ private boolean isAppButtonPresent(String appId) {
     }
 
     public String getLoggedUsername() {
-        return findElement(By.cssSelector(".identifier")).getText();
```

Comment:
Inspection on the element shows there are more than one element using the current value property. The suggested attribute property uniquely identifies the element thus overriding